### DISCUSSION TITLE
Comands to use in graphs LaTeX

### DISCUSSION THREAD

**Discussion by SergeTDU** [xournalpp/xournalpp#5351] [Q&A]
Hi, I wanted to know the possible commands to delimit a function
I was trying to insert an interval in the function it self.
For eg. the interval of my function being [0, +∞[, and not ]-∞,+∞[
![help pls](https://github.com/xournalpp/xournalpp/assets/106892054/986faa98-586d-4213-ba62-e616fb5aaaab)

**Comment by rolandlo** [comment]
The `\graph` command as defined in the [guide](https://xournalpp.github.io/guide/tools/latex/) is a shortcut for
```tex
    \begin{tikzpicture}
    \begin{axis}[xlabel=$x$,ylabel=$y$, axis lines=center,samples=100, #2]
    \addplot[#1]{ #3 };
    \end{axis}
    \end{tikzpicture}
```
Here `#1` stands for what you enter in the first square bracket (e.g. `domain=-1:1,red`), `#2` stands for the second square bracket (e.g. `height=6cm`) and `#3` stands for the function written in curly bracket (e.g. `-x`)

So in the first square bracket you can add any options for the graph, in the second square bracket options for the coordinate system (axis environment).

See [overleaf guide](https://www.overleaf.com/learn/latex/Pgfplots_package#Plotting_mathematical_expressions) for a short survey and [pgfplots manual](https://mirror.init7.net/ctan/graphics/pgf/contrib/pgfplots/doc/pgfplots.pdf) for all the options you can use there.

Specifically to change the interval on which the function is defined to the positive numbers, change `domain=-1:1` to say `domain=0:4`.
